zrml/swaps/src/tests.rs

```diff
@@ -3121,7 +3121,7 @@ fn on_idle_skips_arbitrage_if_price_does_not_exceed_threshold() {
         let pool_id = 0;
         // Force the pool into the cache.
         crate::PoolsCachedForArbitrage::<Runtime>::insert(pool_id, ());
-        Swaps::on_idle(System::block_number(), Weight::max_value());
+        Swaps::on_idle(System::block_number(), Weight::from_ref_time(u64::max_value()));
```

Comment:
```suggestion
        Swaps::on_idle(System::block_number(), Weight::MAX);
```

https://paritytech.github.io/substrate/master/src/sp_weights/weight_v2.rs.html#77